nixos/doc/manual/installation/changing-config.xml

```diff
@@ -54,7 +54,7 @@ from “stable” configurations.</para>
 <para>Finally, you can do
 
 <screen>
-$ nixos-rebuild build</screen>
+# nixos-rebuild build</screen>
```

Comment:
You're right. I tested it and it failed with permission errors, but I forgot that I was in `/root`, which of course is not writable by normal users. m(